src/Sulu/Bundle/ContentBundle/Resources/public/css/main.css

```diff
@@ -189,7 +189,7 @@
   line-height: 32px;
 }
 
-@media only screen and (max-width: 780px) {
+@media only screen and (max-width: 900px) {
```

Comment:
Are these changes really from you? Isn't that work from @marcelmoosbrugger? Maybe this branch is outdated?